apk-generator/v2/scripts/build_apk.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-./gradlew assembleRelease --info
+./gradlew assemble$3Release --info
```

Comment:
@niranjan94 I think in generator we only need release build. `assembleFdroid` & `assembleGoogleplay` will generate debug build which is not used. Also it will take more time.